acloud/resource_cluster.go

```diff
@@ -395,13 +405,13 @@ func getProvider(m interface{}) ConfiguredProvider {
 	return p
 }
 
-func getTransitionStatus(desiredStatus string) string {
+func getTransitionStatus(desiredStatus string) *string {
 	if desiredStatus == string(ClusterStateRunning) {
-		return "starting"
+		return String("starting")
 	} else if desiredStatus == string(ClusterStateStopped) {
-		return "stopping"
+		return String("starting")
```

Comment:
Shouldn't this be "stopping"?
```suggestion
		return String("stopping")
```

Reply:
Thanks!